config.md

```diff
@@ -462,6 +462,12 @@ Instead they MUST ignore unknown properties.
 Runtimes that are reading or processing this configuration file MUST generate an error when invalid or unsupported values are encountered.
 Unless support for a valid value is explicitly required, runtimes MAY choose which subset of the valid values it will support.
 
+## <a name="VirtualMachine" />VM
```

Comment:
The VM-specific section isn't quite platform-specific, but I think this needs to go into the platform-specific section because not all runtimes will be able to support these properties, and platforms are linked to the [compliance language][1].  This is one of the reasons [I prefer “protocol” to “platform”][2].

You should also define the **`vm`** property wherever the Virtual Machine entry ends up in `config.md`.

[1]: https://github.com/opencontainers/runtime-spec/blame/v1.0.1/spec.md#L40-L41
[2]: https://github.com/opencontainers/runtime-spec/pull/570#issue-85434395